masschallenge/opsworks-web-python
# [AC-3840] - EBS Final Cleanup

**Changes Introduced:**

- Add a recipe for adding a symlink from /srv/www/mc to /vol/www/mc, to be used on testing machines that have to small root device.

**Test:**
- Go to a test machine other than test 4.
- Stack -> Settings, set the opsworks-web-python branch to AC-3840
- Deployments -> Run Command -> Update Custom Cookbooks.
- Stack -> Layers -> MassChallenge -> EBS Volumes , set it to be like [test 4's](https://console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/67a723c8-ea0a-49ce-a7ad-45427de81248/layers/67100ed3-96d5-437f-bcb8-d4f8da04b580/volumes/edit)
- Stack -> Layers -> MassChallenge -> Recipes, add to "Setup" the custom recipe "opsworks_deploy_python::symlink-volume". Should look like this: 
![image](https://user-images.githubusercontent.com/5147457/39522566-47fde50e-4e1b-11e8-8384-d016694f4601.png)
- Stack -> Instances, add another instance. Choose m3.medium, give it a sensible name, other settings should remain the same.
- Start the machine, wait for setup to end.
- ssh into that new instance. run `cd /srv/www;ll;`, you'll see that there's a symlink to the loaded volume.
